app/main.py

```diff
@@ -1,3 +1,17 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: any) -> any:
+    cached_list = {}
+
+    def inner(*args) -> any:
+        if args in cached_list:
+
+            cached_list[args] = "Getting from cache"
+            print("Getting from cache")
+
+            for item in args:
```

Comment:
why do you iterate over args? 
you should return a result, not an argument

Reply:
When I trying return a result I have a error:   FAILED tests/test_main.py::test_cache_returns_correct_value - assert (2, 3, 3) == 2.  But when I return a argument all is passed. Why?